hc/accounts/tests/test_login.py

```diff
@@ -18,20 +18,25 @@ def test_it_sends_link(self):
 
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
-
-        ### Assert that a user was created
+        # Assert that a user was created
+        self.assertEqual("alice@example.org", User.objects.get(pk=1).email)
```

Comment:
@daisymacharia 

The tests for postgres and mysql are failing at this point.

Instead of hardcoding the id (`pk=1`) try getting the user id in this manner;

- query all the users; `User.objects.all()` (this should return a total of only one user at this point)
- access index 0 from the operation above to get the single user
- pass in that user's id in place of the hardcoded value 1 i.e. (`pk=user.id`)